internal/sidekick/sidekick/downloads_cache.go

```diff
@@ -76,7 +75,11 @@ func makeSourceRoot(rootConfig *config.Config, configPrefix string) (string, err
 func extractTarball(source, destination string) error {
 	cmd := exec.Command("tar", "-zxf", source)
 	cmd.Dir = destination
```

Comment:
nit: could maybe use `Run(...)`, if you add `"-C", destination` to the command, but no big deal